guide/sequelize/currency.md

```diff
@@ -362,6 +364,20 @@ return message.channel.send(
 	{ code: true }
 );
 ```
+</branch>
```

Comment:
Very small nit-pick, but to keep consistency with [CONTRIBUTING.md](https://github.com/discordjs/guide/blob/master/CONTRIBUTING.md) and since we had issues with non-spaced branch tags before: please keep an empty line between branch tags and any other content.